source/private.rst

```diff
@@ -21,10 +21,9 @@ backups involves following steps:
            --set accessKey=some-access-key \
            --set secretKey=some-secret-key \
            --set service.type=ClusterIP \
-           --set configPath=/tmp/.minio/ \
+           --set configPath='~/.minio' \
```

Comment:
is this path works fine on OpenShift?

Reply:
Yeap. I made such a change in order to make this command work on Win + git bash. '/tmp/.minio' path is reconstructed into something like 'C:\Users\...' by git bash which leads to startup failure.

Reply:
please test it on real OpenShift